### DISCUSSION TITLE
prometheus error

### DISCUSSION THREAD

**Discussion by Rajesh-2406** [learndevopsonline/learndevopsonline#1451] [Q&A] [answered]
![Screenshot 2023-09-15 191453](https://github.com/learndevopsonline/learndevopsonline/assets/136773140/6c48b09e-6022-4c69-8f07-641e1f3dec41)
prometheus id - 54.205.120.35

**Comment by sreedharm07** [comment, marked as answer]
Hi, @Rajesh-2406 
i just installed prometheus (labauto prometheus)in your server. now its working fine.
may be you dint installed prometheus properly or i think you installed wrong one thats why its not worked for you.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/d3f622c0-a39e-4421-9478-adc7438ea723)
